src/content/docs/distribute/Signing/android.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Android Code Signing
+sidebar:
+  label: Android
+  order: 5
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Code } from '@astrojs/starlight/components';
+
+To publish on the Play Store, you need to sign your app with a digital certificate.
+
+Android App Bundles and APKs must be signed before being uploaded for distribution.
+
+Google also provides an additional signing mechanism for Android App Bundles distributed in the Play Store.
+See the [official Play App Signing documentation] for more information.
+
+## Creating a keystore and upload key
+
+Android signing requires a Java Keystore file that can be generated using the official `keytool` CLI:
+
+<Tabs>
+  <TabItem label="macOS / Linux">
+  
+```
+keytool -genkey -v -keystore ~/upload-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias upload
+```
+
+  </TabItem>
+
+  <TabItem label="Windows">
+  
+```
+keytool -genkey -v -keystore $env:USERPROFILE\upload-keystore.jks -storetype JKS -keyalg RSA -keysize 2048 -validity 10000 -alias upload
+```
+
+  </TabItem>
+</Tabs>
+
+This command stores the `upload-keystore.jks` file in your home directory.
+If you want to store it elsewhere, change the argument you pass to the `-keystore` parameter.
+
+:::note
+
+- The `keytool` command might not be in your PATH.
+  You may find it installed in the JDK that is installed with Android Studio:
+
+<Tabs>
+
+<TabItem label="Linux">
+  <Code code="/opt/android-studio/jbr/bin/keytool ...args" lang="sh" />
+  **Android Studio directory path depends on your Linux distribution**
+</TabItem>
+
+<TabItem label="macOS">
+  <Code
+    code="/Applications/Android\ Studio.app/Contents/jbr/Contents/Home/bin/keytool ...args"
+    lang="sh"
+  />
+</TabItem>
+
+<TabItem label="Windows">
+  <Code
+    code="C:\\Program Files\\Android\\Android Studio\\jbr\\bin\\keytool.exe ...args"
+    lang="sh"
+  />
+</TabItem>
+
+</Tabs>
+
+:::
+
+:::caution
+
+Keep the `keystore` file private; don't check it into public source control!
+
+:::
+
+See the [official documentation](https://developer.android.com/studio/publish/app-signing#generate-key) for more information.
+
+## Configure the signing key
+
+Create a file named `[project]/src-tauri/gen/android/keystore.properties` that contains a reference to your keystore:
+
+```
+password=<password defined when keytool was executed>
+keyAlias=upload
+storeFile=<location of the key store file, such as /Users/<user name>/upload-keystore.jks or C:\\Users\\<user name>\\upload-keystore.jks>
+```
+
+:::caution
```

Comment:
```suggestion
:::caution[Security Warning]
```

Also feel having two security warnings on the same page could be condensed into one as it is about keeping secrets private.